zerver/views/auth.py

```diff
@@ -151,6 +151,7 @@ def maybe_send_to_registration(request: HttpRequest, email: str, full_name: str=
             # was created.
             prereg_user.password_required = password_required
             update_fields = ["password_required"]
+            print("aquiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiii", prereg_user)
```

Comment:
Left a debug print here :octocat: 
Looks like the right first step, though perhaps we should call it something like  `user_avatar_url` to make it clear it's an url, rather than the actual image content?